setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
accidental commit of this file?

Reply:
Yes, was checking the Nix version, removed!